<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>886</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="Title">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>102</y>
      <width>341</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe Devanagari</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>OD METER Ying ver.2.1</string>
    </property>
   </widget>
   <widget class="QLabel" name="ODlabel">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>420</y>
      <width>81</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>OD</string>
    </property>
   </widget>
   <widget class="QLabel" name="gamma12label">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>620</y>
      <width>81</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>gamma12</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="OD">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>440</y>
      <width>181</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe Devanagari</family>
      <pointsize>24</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="gamma12">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>640</y>
      <width>181</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe Devanagari</family>
      <pointsize>24</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>150</y>
      <width>621</width>
      <height>251</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="graph_layout"/>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>410</y>
      <width>591</width>
      <height>271</height>
     </rect>
    </property>
    <property name="title">
     <string>Fitting Options</string>
    </property>
    <widget class="QCheckBox" name="enableFitting">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>161</width>
       <height>22</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Enable Fitting</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>30</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="editable">
      <bool>false</bool>
     </property>
     <item>
      <property name="text">
       <string>Two-level</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>EIT</string>
      </property>
     </item>
    </widget>
    <widget class="QCheckBox" name="enhancedFitting">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>200</y>
       <width>231</width>
       <height>21</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Enhanced EIT Fitting</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QSpinBox" name="enhancedPara">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>200</y>
       <width>49</width>
       <height>25</height>
      </rect>
     </property>
     <property name="minimum">
      <number>2</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="smoothPara">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>230</y>
       <width>49</width>
       <height>25</height>
      </rect>
     </property>
     <property name="minimum">
      <number>2</number>
     </property>
    </widget>
    <widget class="QCheckBox" name="smoothFitting">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>230</y>
       <width>231</width>
       <height>21</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Smooth Fitting</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>571</width>
       <height>27</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="Offset"/>
      </item>
      <item>
       <widget class="QLineEdit" name="Amplitude"/>
      </item>
      <item>
       <widget class="QLineEdit" name="OD_3"/>
      </item>
      <item>
       <widget class="QLineEdit" name="Delta_p"/>
      </item>
      <item>
       <widget class="QLineEdit" name="gamma_12"/>
      </item>
      <item>
       <widget class="QLineEdit" name="Omega_c"/>
      </item>
      <item>
       <widget class="QLineEdit" name="Delta_c"/>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="Paras">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>59</y>
       <width>513</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Initial fitting parameters &amp; Real-time fitting parameters</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>571</width>
       <height>27</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLineEdit" name="Offset_2">
        <property name="enabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="Amplitude_2"/>
      </item>
      <item>
       <widget class="QLineEdit" name="OD_2"/>
      </item>
      <item>
       <widget class="QLineEdit" name="Delta_p_2"/>
      </item>
      <item>
       <widget class="QLineEdit" name="gamma12_2"/>
      </item>
      <item>
       <widget class="QLineEdit" name="Omega_c_2"/>
      </item>
      <item>
       <widget class="QLineEdit" name="Delta_c_2"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>584</width>
       <height>24</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QCheckBox" name="enableOffset">
        <property name="text">
         <string>Offset</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="enableAmp">
        <property name="text">
         <string> Amp</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="enableOD">
        <property name="text">
         <string>  OD</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="enableDelta_p">
        <property name="text">
         <string>Δ_p</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="enableGamma12">
        <property name="text">
         <string>γ_12</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="enableOmega_c">
        <property name="text">
         <string> Ω_c</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="enableDelta_c">
        <property name="text">
         <string> Δ_c</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>680</y>
      <width>591</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>Save Options</string>
    </property>
    <widget class="QLineEdit" name="saveStatus">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>571</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="fileIndex">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>50</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="fileDirectory">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>50</y>
       <width>301</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QCheckBox" name="autoIndex">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>50</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="fileName">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>50</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="fileDirectory_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>651</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Auto-index File Directory                    Name/Prefix From</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>150</y>
      <width>157</width>
      <height>251</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="bg">
       <property name="text">
        <string>Take Background</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="clearBg">
       <property name="font">
        <font>
         <weight>50</weight>
         <bold>false</bold>
         <kerning>false</kerning>
        </font>
       </property>
       <property name="text">
        <string>Clear Background</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="data">
       <property name="text">
        <string>Take Data</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="single">
       <property name="text">
        <string>Single Frame</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Save">
       <property name="text">
        <string>Save Data</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="STOP">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>STOP</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>510</y>
      <width>181</width>
      <height>101</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="graph_layout_2"/>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>710</y>
      <width>181</width>
      <height>101</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="graph_layout_3"/>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menupy">
    <property name="title">
     <string>OD</string>
    </property>
   </widget>
   <addaction name="menupy"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
